### PR TITLE
Fix upload_artifact script to support rebuild of bottle  Fix #128

### LINKED ISSUE #128: brew install hang (use source) -> missing iblinter-0.4.20.mojave.bottle.tar.gz
I use iblinter in github workflow/action
2 minutes to install
So I look to the log and see that there is no iblinter-0.4.20.mojave.bottle.tar.gz attached for this release
https://github.com/IBDecodable/IBLinter/releases/tag/0.4.20
```
Run brew install IBDecodable/homebrew-tap/iblinter
==> Tapping ibdecodable/tap
Cloning into '/usr/local/Homebrew/Library/Taps/ibdecodable/homebrew-tap'...
Tapped 1 formula (32 files, 26.6KB).
==> Installing iblinter from ibdecodable/tap
==> Downloading https://github.com/IBDecodable/IBLinter/releases/download/0.4.20/iblinter-0.4.20.mojave.bottle.tar.gz
curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "iblinter"
Download failed: https://github.com/IBDecodable/IBLinter/releases/download/0.4.20/iblinter-0.4.20.mojave.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Cloning https://github.com/IBDecodable/IBLinter.git
Cloning into '/Users/runner/Library/Caches/Homebrew/iblinter--git'...
==> Checking out tag 0.4.20
HEAD is now at 4b518da Merge pull request #125 from IBDecodable/release/0.4.20
==> make install PREFIX=/usr/local/Cellar/iblinter/0.4.20
🍺  /usr/local/Cellar/iblinter/0.4.20: 6 files, 8.9MB, built in 2 minutes 1 second
```

So how to build it? 
Maybe https://github.com/IBDecodable/IBLinter/blob/master/.github/workflows/artifact.yml#L40
and the workflow failed here https://github.com/IBDecodable/IBLinter/commit/4b518dafbd64a329374f285c5f27bc6c08be7a38/checks?check_suite_id=257732098#step:5:217

### PR DESCRIPTION
**Pull request by kateinoigakukun**
Fix upload_artifact script to support rebuild of bottle

### CODE REVIEW

**Comment by phimage** [comment]
This Fix #128 ? if yes 🙏
we could rename PR with Fix #128 to automatically close the issue

**Comment by kateinoigakukun** [comment]
Ah, wait a minutes to merge

**Comment by kateinoigakukun** [comment]
Updated

File: docs/Release.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+$ git checkout -b release/0.4.20
+$ make bump_version 0.4.20
+$ vim CHANGELOG.md # Update CHANGELOG.md
+$ git push origin HEAD
+```
+
+5. After merged release PR, tag and push the new version
+6. Wait `artifact` CI job which update CocoaPods version, Homebrew bottle, and Homebrew tap
+7. Completed :tada:
```
Comment (phimage) [inline docs/Release.md]: :tada: or unicode 🎉 🙃

**Comment by phimage** [comment]
github workflow is so cool that we want to do all with it 😆

**Pull request merged**